### ISSUE TITLE
enableEdgeToEdge hides EditText/TextField on MessagesScreen [Compose]

### ISSUE THREAD

**Issue by mtali** [GetStream/stream-chat-android#5304]
**Describe the bug**  
When `edgeToEdge` is enabled, the keyboard hides the input `EditText/TextField` in `MessageActivity`, making it difficult for users to see what they are typing.

**SDK version**  
- 6.4.0 (Stream Chat SDK)

**To Reproduce**  
1. Go to `MessageActivity`.
2. Enable `edgeToEdge` in the `onCreate` method.
3. Focus on the `EditText/TextField` to bring up the keyboard.
4. Observe that the keyboard hides the input field.

**Expected behavior**  
When the keyboard is displayed, the `EditText/TextField` should remain visible, adjusting the layout so that users can see what they are typing.

**Device:**  
- Vendor and model: Oppo CPH2461
- Android version: 14

**Code Snippet**

```kotlin
class MessageActivity : ComponentActivity() {
  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    enableEdgeToEdge() // <------------------- REMOVING THIS FIXES THE ISSUE BUT I NEED IT
    val extras = checkNotNull(intent.extras)
    val channelId = extras.getString(CHANNEL_ID_KEY) ?: error("Null channelId")

    setContent {
      ChatTheme {
        Scaffold { innerPadding ->
          Box(modifier = Modifier.padding(innerPadding)) {
            MessagesScreen(
              viewModelFactory = MessagesViewModelFactory(
                context = this@MessageActivity,
                channelId = channelId,
                messageLimit = 30,
              ),
              onBackPressed = { onBackPressedDispatcher.onBackPressed() },
            )
          }
        }
      }
    }
  }

  companion object {
    private const val CHANNEL_ID_KEY = "channelId"

    fun launch(context: Context, channelId: String) {
      val intent = Intent(context, MessageActivity::class.java)
      intent.putExtra(CHANNEL_ID_KEY, channelId)
      context.startActivity(intent)
    }
  }
}
```

**AndroidManifest.xml**

```xml
<activity
  android:name=".chat.MessageActivity"
  android:exported="false"
  android:windowSoftInputMode="adjustResize"
/>
```

**Build Configuration**

```groovy
const val COMPILE_SDK = 34
const val TARGET_SDK = 34
const val MIN_SDK = 27
```

**Screenshots**  

- This is when you have enableEdgeToEdge() enabled 

![WhatsApp Image 2024-06-23 at 23 45 52](https://github.com/GetStream/stream-chat-android/assets/29874428/37733c59-1b30-4bb0-8078-b7a5a2850f0b)

- This is when you don't have enableEdgeToEdge()

![WhatsApp Image 2024-06-23 at 23 49 29](https://github.com/GetStream/stream-chat-android/assets/29874428/d6c07be2-d3ab-4b88-9a66-ad61bc865023)

**Comment by mtali** [comment]
I have also tried adding enableEdgeToEdge() on `BaseConnectedActivity` on [stream-chat-android-compose-sample](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-compose-sample/src/main/java/io/getstream/chat/android/compose/sample/ui/BaseConnectedActivity.kt), same thing happens

**Comment by aleksandar-apostolov** [comment]
Hey @mtali thank you for the report, we already looking into this, will keep you posted.

**Comment by mtali** [comment]
> Hey @mtali thank you for the report, we already looking into this, will keep you posted.

**Comment by mtali** [comment]
Hello @aleksandar-apostolov , thanks for your response. After some digging i think the solution is to add  .imePadding() on MessageInput. I think it is also a good idea to add some safe padding on MessageInput (Maybe )

```
MessageInput(
        modifier = Modifier
          .fillMaxWidth()
          .imePadding()
          .weight(7f)
          .padding(start = 8.dp)
```

**Comment by kanat** [comment]
Hi @mtali,

This issue was addressed in the following PR:
- https://github.com/GetStream/stream-chat-android/pull/5386

It will be available with the next upcoming release 👍

**Comment by kanat** [comment]
Fixed in [v6.5.0](https://github.com/GetStream/stream-chat-android/releases/tag/v6.5.0)
